granary/reddit.py

```diff
@@ -0,0 +1,215 @@
+# coding=utf-8
+"""reddit.com source class.
+"""
+
+from . import source
+import datetime
+import logging
+from oauth_dropins import reddit
+from oauth_dropins.webutil import util
+import re
+import urllib.parse, urllib.request
+
+import praw
+
+class Reddit(source.Source):
+
+    DOMAIN = 'reddit.com'
+    BASE_URL = 'https://reddit.com'
+    NAME = 'reddit'
+
+    def __init__(self, refresh_token):
+        self.refresh_token = refresh_token
+
+    @staticmethod
+    def timestamp_to_iso8601(time_stmp):
```

Comment:
ooh, good timing, i actually added this as `webutil.util.maybe_timestamp_to_iso8601()` just recently!

Reply:
cool, and i hadn't realized that webutils was a sub repo (or whatever that's called)